resources/index.scss

```diff
@@ -303,6 +303,35 @@ button.as-link {
     }
 }
 
+// Remove once framework is released
```

Comment:
is there a corresponding issue in the framework project?

Reply:
Yes it is merged and will be released soon